tempodb/backend/raw.go

```diff
@@ -113,6 +113,19 @@ func NewReader(r RawReader) Reader {
 	}
 }
 
+func (r *reader) Has(ctx context.Context, name string, blockID uuid.UUID, tenantID string) (bool, error) {
+	objects, err := r.r.List(ctx, KeyPathForBlock(blockID, tenantID))
```

Comment:
No idea if this works for all backends :). We only really need it to work for local, but that's not ideal.